selenium.go

```diff
@@ -370,6 +370,10 @@ type WebDriver interface {
 	// perform JSON decoding.
 	ExecuteScriptAsyncRaw(script string, args []interface{}) ([]byte, error)
 
+	// ExecuteCdpCommand execute Chrome Devtools Protocol command and get returned result
```

Comment:
Rephrase documentation comment as:

// ExecuteChromeDPCommand executes a Chrome DevTools Protocol command.
//
// See https://chromedevtools.github.io/devtools-protocol/ for available commands.